composables/album.ts

```diff
@@ -1,25 +1,17 @@
-import { AlbumApi, AlbumModel } from "@bcc-code/bmm-sdk-fetch";
+import { AlbumApi } from "@bcc-code/bmm-sdk-fetch";
 
 interface UseAlbumOptions {
   id: number;
 }
 
-/**
- * Get playlist with the specified id
- */
 export function useAlbum(options: UseAlbumOptions) {
   const { id } = options;
 
-  return useLazyAsyncData<AlbumModel>(`album-${id}`, () =>
+  return useLazyAsyncData(`album-${id}`, () =>
```

Comment:
Are the types (AlbumModel) inferred here?

Reply:
Yes, they are 👍